arm-web/2015-08-01-preview/swagger/logicAppsManagementClient.json

```diff
@@ -0,0 +1,1281 @@
+          "description": "Version of product",
+          "type": "string"
+        }
+      }
+    },
+    "SkuDescription": {
+      "description": "Describes a sku for a scalable resource",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the resource sku",
+          "type": "string"
+        },
+        "tier": {
+          "description": "Service Tier of the resource sku",
+          "type": "string"
+        },
+        "size": {
+          "description": "Size specifier of the resource sku",
+          "type": "string"
+        },
+        "family": {
+          "description": "Family code of the resource sku",
+          "type": "string"
+        },
+        "capacity": {
+          "format": "int32",
+          "description": "Current number of instances assigned to the resource",
+          "type": "integer"
+        }
+      }
+    },
+    "ConsentLinkInput": {
+      "description": "Connection Constent Link payload",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "properties": {
+            "parameters": {
+              "description": "Array of links",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ConsentLinkInputParameter"
+              }
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ConsentLinkInputParameter": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "principalType": {
+          "description": "Principal type",
+          "enum": [
+            "ActiveDirectory",
+            "Connection",
+            "MicrosoftAccount"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PrincipalType",
+            "modelAsString": false
+          }
+        },
+        "tenantId": {
+          "description": "Tenant Id",
+          "type": "string"
+        },
+        "objectId": {
+          "description": "AAD OID (user or group) if the principal type is ActiveDirectory.\r\n            MSA PUID if the principal type is MicrosoftAccount.",
+          "type": "string"
+        },
+        "parameterName": {
+          "description": "Name of the parameter in the connection provider's oauthSettings",
+          "type": "string"
+        },
+        "redirectUrl": {
+          "description": "Name of the parameter in the connection provider's oauthSettings",
+          "type": "string"
+        }
+      }
+    },
+    "ConsentLinkPayload": {
+      "description": "Collection of consent links",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConsentLink"
+          }
+        }
+      }
+    },
+    "ConsentLink": {
```

Comment:
Can we name this listConsentLinksInput to be consistent with other definition names for payloads?